docs/Apps/Metabolomic Data/Untargeted Pipeline.md

```diff
@@ -192,7 +192,255 @@ The results are generated in the MetScape input intensity format and metadata fo
 
 ![MetScape cohort file](../../img/UntargetedPipeline/MetScapeCohort.png)<center>**Figure 18.** MetScape cohort file</center>
 
+##GCT-Preparation
 
+This interface allows user to select the data to be used in the downstream pipeline. The user can use any data to the downstream analysis by selecting the data from the previous steps.
+
+![GCT-Preparation Tab](../../img/UntargetedPipeline/GCTPreparation.png)<center>**Figure 19.** GCT-Preparation Tab</center>
+
+*   *Uploaded data:* This allows you to take the uploaded data to the downsteam pipeline.
+*   *Annotated Data:* This allows you to take the annotated data to the downsteam pipeline. The user can take the representative data or the overall data to the downsteam pipeline.
+*   *Identified data:* This allows you to take the identified data to the downsteam pipeline. The user can take the representative data or the overall data to the downsteam pipeline.
```

Comment:
user--> you, can "either" take the representative data or..., add above as well

Reply:
Done